sroka/api/ga/README.md

```diff
@@ -37,3 +37,37 @@ request = {
 df_ga = ga_request(request, print_sample_size=True, sampling_level='FASTER')
 
 ```
+
+
+### ga_request_all_data(request_parameters, start_index, page_size, max_pages, print_sample_size, sampling_level):
+
+#### Arguments
+
+* request_parameters (obligatory) - the dictionary of GA request parameters
+* start_index - the index of the first element to be retrieved (optional, default = 1)
+* page_size - the number of elements retrieved in a single request (optional, default = 10000)
+* max_pages - the max number of pages to retrieve, None if all available pages (optional, default = None)
+* print_sample_size - if True, prints the sample size of every request (optional, default = False)
+* sampling_level - the GA sampling level (optional, default = HIGHER_PRECISION)
```

Comment:
We could add a check if sampling_level is within 'DEFAULT', 'FASTER', 'HIGHER_PRECISION' as it also throws an error if different value is provided. I know this was not handled earlier but this is an opportunity to do it better.